### ISSUE TITLE
wrong Rhea on GO:0005381

### ISSUE THREAD

**Issue by rwst** [geneontology/go-ontology#21441]
GO:0005381 has an xref to RHEA:29579 which is about iron(2+) transport. However there seems no Rhea reaction that explicitly uses CHEBI:24875 (iron cation) and the search for CHEBI:24875 in Rhea yields all reactions with subclasses of CHEBI:24875.

**Comment by pgaudet** [comment]
Right, RHEA:29579 describes a Fe2+/H+ symporter - which looks like it corresponds to one of the activities of http://www.tcdb.org/search/result.php?tc=2.A.55.2.1
Obsolete GO:0015335 obsolete heavy metal ion:hydrogen symporter activity was cross-referencing this term. 
This corresponds to P49281 SLC11A2 Natural resistance-associated macrophage protein 2

I removed the cross reference. 

===

However this touches on GO term specificity. For transport, I think we are moving towards describing broad specificity transporters, so perhaps restoring GO:0015335 (and renaming it as in TCDB 'divalent heavy metal ion:H+ symporter' would be appropriate. 

I'll keep this open for editors's discussion. 
Thanks, Pascale

**Comment by alanbridge** [comment]
Hi Pascale, 

can I suggest you set the specificity of transporters in GO at the same level as the specificity of enzymes in GO?

So for example for lipids, transport of lipid classes is in (e.g. TAG), while transport of specific structural variants of those classes, that include particular combinations of fatty acids, is out (e.g. TAG(16:0/16:0/18:1)). The latter is one of tens of thousands of possibilities.

Perhaps a good rule of thumb would be that if the molecule being transported appears in a GO MF enzyme term already, then one might reasonably need to annotate the specific transporters that transport it around the cell (certainly the case for Fe2+ as revealed by a quick search in Amigo: http://amigo.geneontology.org/amigo/search/ontology?q=%20Fe2+). 

All the best, Alan

**Comment by pgaudet** [comment]
Talking with @thomaspd 
Given the recent discussions we've had about specificity, would it be possible to create a more general Rhea reaction using CHEBI:60240 divalent metal cation ? This would allow a precise mapping between the GO term and Rhea. 

Thanks, Pascale
